### PR TITLE
Hexal-PR

### PR DESCRIPTION
**Pull request by Ildorado**

### CODE REVIEW

**Comment by verus774** [comment]
## Fines:
* missing requirements (interactive nav, no `form` tag and as result non-working fields validation): -5
* no pixel perfect in "about" section, wrong lines sizes and colors in some sections: -5
* no "assets" folder, mix camel and kebab cases in classes naming, extra closing `div` tags, `alt` attributes for `div` : -5
## Total: 85

File: index.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" media="screen" href="styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="nav">
+            <ul>
+                <li><a href="#portfolio" class="portfolio-nav-link">PORTFOLIO</a></li>
+                <li class="dot-li"><span class="dot"></span></li>
```
Comment (mrsDandelion) [inline index.html:20]: More proper, if you use selector li:not(::last-child)::after {...} for creating dot

File: index.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                <li><a href="#about" class="nav-link">ABOUT</a></li>
+                <li ><img src="Logo@1X.png" alt="Logo" class="logo"></li>
+                <li><a href="#" class="nav-link">BLOG</a></li>
+                <li class="dot-li"><span class="dot"></span></li>
+                <li><a href="#getInTouch" class="nav-link">GET IN TOUCH</a></li>
+            </ul>
+        </div>
+    </header>
+    <main>
+        <section id="portfolio" class="">
```
Comment (mrsDandelion) [inline index.html:30]: Why do you have empty class?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                        <a href="#" class="fa fa-dribbble"></a>
+                    </div>
+                </article>
+            </div>
+        </section>
+
+    </main>
+    <footer>
+        <section id="getInTouch" class="getInTouch-section">
+            <h1>Get In Touch</h1>
```
Comment (mrsDandelion) [inline index.html:157]: You can't have two h1. Only one

File: index.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            <h2 class="color-gray-for-h2">In order to get in touch use the contact form below:</h2>
+            <div class="getInTouch">
+                <div class="form">
+                    <input type="text" name="name" id="name" class="input" placeholder="Name(Required)" required>
+                    <input type="text" name="email" id="email" class="input" placeholder="Email(Required)" required>
```
Comment (mrsDandelion) [inline index.html:162]: Use <input type="email"...>

File: index.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                    <h2 class="color-green-for-h2">Illustrator & Character designer</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                        Mauris porttitor, nibh at sagittis tristique,
+                        enim magna pretium augue, eget consectetur sem dui non urna.
+                        Integer gravida, diam congue molestie fringilla,
+                        elit odio rhoncus turpis, ac dictum libero elit vel metus.
+                        Morbi egestas sit amet velit eget venenatis.
+                        Mauris porttitor, nibh at sagittis tristique,
+                        enim magna pretium augue, eget consectetur sem dui non urna</p>
+                    <div class="social-networks-links">
```
Comment (mrsDandelion) [inline index.html:142]: Use ul with li

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+    text-align: center;
+    position: relative;
+}
+
+header:after {
+    content: "";
+    width: 0;
+    height: 0;
+    z-index: 2;
+    background-color: #ffffff;
```
Comment (mrsDandelion) [inline styles.css:99]: You can use only #fff

File: index.html [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+
+            </div>
+
+
+            </div>
+        </section>
+        <section id="about" class="about-section">
+            <div class="about">
+                <div>
+                        <div class="hexagon-1" alt="Smith Anderson" style="background-image: url('SmithAnderson.png');">
```
Comment (verus774) [inline index.html:125]: try to avoid inline styles

**Review by verus774 — APPROVED** [review]

**Pull request closed without merging**